src/main/java/aquality/selenium/core/elements/Element.java

```diff
@@ -36,7 +36,7 @@ protected Element(final By loc, final String name, final ElementState state) {
 
     protected abstract IElementCacheConfiguration getElementCacheConfiguration();
 
-    protected abstract IElementActionRetrier getElementActionRetrier();
+    protected abstract IActionRetrier getActionRetrier();
```

Comment:
I believe this should remain as IElementActionRetrier

Reply:
updated